packages/swapper/src/swappers/RelaySwapper/endpoints.ts

```diff
@@ -0,0 +1,178 @@
+import { fromChainId } from '@shapeshiftoss/caip'
+import { TxStatus } from '@shapeshiftoss/unchained-client'
+import { bnOrZero } from '@shapeshiftoss/utils'
+import type { Result } from '@sniptt/monads/build'
+import { Err } from '@sniptt/monads/build'
+import type { InterpolationOptions } from 'node-polyglot'
+
+import type {
+  CommonTradeQuoteInput,
+  EvmTransactionRequest,
+  GetTradeRateInput,
+  GetUnsignedEvmTransactionArgs,
+  SwapErrorRight,
+  SwapperApi,
+  SwapperDeps,
+  TradeQuote,
+  TradeRate,
+} from '../../types'
+import { TradeQuoteError } from '../../types'
+import { checkSafeTransactionStatus, isExecutableTradeQuote, makeSwapErrorRight } from '../../utils'
+import { relayChainMap } from './constant'
+import { getTradeQuote } from './getTradeQuote/getTradeQuote'
+import { getTradeRate } from './getTradeRate/getTradeRate'
+import { relayService } from './utils/relayService'
+import type { RelayStatus } from './utils/types'
+
+export const relayApi: SwapperApi = {
+  getTradeQuote: async (
+    input: CommonTradeQuoteInput,
+    deps: SwapperDeps,
+  ): Promise<Result<TradeQuote[], SwapErrorRight>> => {
+    if (input.sellAmountIncludingProtocolFeesCryptoBaseUnit === '0') {
+      return Err(
+        makeSwapErrorRight({
+          message: 'sell amount too low',
+          code: TradeQuoteError.SellAmountBelowMinimum,
+        }),
+      )
+    }
```

Comment:
I know this is copied over from Li.Fi but we don't need this. Quotes with 0 won't fire, and no other swapper does it, this was added as a paranoia and isn't hit.

Reply:
Removed in [29fc44d](https://github.com/shapeshift/web/pull/9188/commits/29fc44d8ce158086f2cbec2c3c7ee0dd44302dac)